### PR TITLE
ISSUE #3160 - Change styling of 'Add new Federation/Container' at the bottom of lists

### LINKED ISSUE #3160: Change styling of 'Add new Federation/Container' at the bottom of lists
On the basis, that the previous design was far too bright and distracting, can we **please** change the style of this component globally.

**Build**
image

**Updated design**
image
Background-color: New hex (#E8EEF8)
Text and icon colour: secondary.main (#6B778C)

**Updated hover state**
image
Change text and icon colour to primary.main (#172B4D)

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3160 

#### Description
- Change the styling to make the buttons less bright and distracting

#### Test cases
- hover over button.
- click and hold button to see active state is the same as hover

### CODE REVIEW

File: frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts [hunk 1]
```diff
@@ -50,25 +50,25 @@ export const Divider = styled(DividerComponent)`
...
 
 export const DashboardListButton = styled(Button).attrs({
-	variant: 'outlined',
-	color: 'primary',
+	variant: 'contained',
+	color: 'secondary',
 })`
 	${({ theme }) => theme.typography.h3}
+	/* TODO - fix when new palette is released */
+	background-color: #E8EEF8;
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts:58]: I'snt the background colour just the primary colour with a different opacity? (consult with @george-hadfield) 
in that case we can have a helper funtion for changing the opacity to a colour.

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts:58]: The issue written up by George says "_Background-color: New hex (#E8EEF8)_"

Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts:58]: I think you should re consult with him about it. I think he is just colour picking the colour and not thinking about what it actually means.

Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts:58]: And if the case is that he actually wants a solid colour we should add it to the palette now, so you would need to consult with George again about what it actually means and if he has the concept of that colour in a different palette

**Comment by george-hadfield** [comment]
@The-Daniel really sorry to mess you around on this one but can I request one small change:

Make the text and icon use the primary teal by default and make the hover state for the text and icon the same colour as the hover state for the button.

**Comment by The-Daniel** [comment]
> @The-Daniel really sorry to mess you around on this one but can I request one small change:
> 
> Make the text and icon use the primary teal by default and make the hover state for the text and icon the same colour as the hover state for the button.

No worries, but I don't quite understand. The button colour doesn't change on hover. And am I right in thinking you want the button/text/icon to all be the same colour on hover so you can't read the 'Add Federation' text?

**Comment by george-hadfield** [comment]
The button colour will not change on hover. The text and icon should be primary.main by default. and if you hover anywhere on the button, the text and icon should go a shade darker to primary.dark.

**Review by george-hadfield — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**